src/Symfony/Component/AssetMapper/Compressor/SupportedCompressorInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Compressor;
+
+/**
+ * @internal
+ *
+ * @author Kévin Dunglas <kevin@dunglas.dev>
+ */
+interface SupportedCompressorInterface extends CompressorInterface
```

Comment:
`SupportableCompressorInterface`? Here I kind of understand that the compressor is always supported.